### DISCUSSION TITLE
Multiple open positions

### DISCUSSION THREAD

**Discussion by kosi96** [kernc/backtesting.py#255] [Q&A] [answered]
Hello,

I am wondering if it is possible to have multiple open positions?

**Comment by kernc** [comment, marked as answer]
Yes, as long as you don't pass [`Backtest(..., exclusive_orders=True)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest) (and, maybe, [`hedging=False`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest)), every [`Strategy.buy()/.sell()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.buy) call places an order that results in a separate [trade](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.trades). The trades, however, still sum up to a single [net total position](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Position).

**Reply by kosi96** [reply]
So where can I whole list of open positions? There is only one listed in `self.position`.

**Reply by kernc** [reply]
:point_right: [`Strategy.trades`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.trades)

The nomenclature follows that of a popular broker.

**Reply by cruizytrader** [reply]
Do you have a code example/sample to trade maximum 3 open positions for the same instrument?
I print Strategy.trades and get "<property object at 0x0000029F117CE400>". I cannot pull out the number of open positions.
Please assist.

**Reply by kernc** [reply]
@cruizytrader `.trades` is a property upon an _instance_ of `Strategy` (namely `self`).

```py
if len(self.trades) < 3:
    self.buy()
```

**Reply by cruizytrader** [reply]
Thanks
